odxtools/positioneddataobjectproperty.py

```diff
@@ -1,78 +0,0 @@
-# SPDX-License-Identifier: MIT
-from dataclasses import dataclass
-from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
-from xml.etree import ElementTree
-
-from .dataobjectproperty import DataObjectProperty
-from .decodestate import DecodeState
-from .encodestate import EncodeState
-from .exceptions import odxrequire
-from .odxlink import OdxDocFragment, OdxLinkDatabase, OdxLinkId, OdxLinkRef
-from .odxtypes import ParameterValue
-
-if TYPE_CHECKING:
-    from .diaglayer import DiagLayer
-
-
-@dataclass
-class PositionedDataObjectProperty:
```

Comment:
why? why? why?

Reply:
because `PositionedDataObjectProperty` is not a `DataObjectProperty`, it does not appear in the XSD, and it is only used in two places where for each occurrence about of 10 lines of code are required to replace it. (I have concluded that it is best for our sanity to match the hierarchy of python classes as closely with the one defined by the XSD as possible. Work in progress...)